lib/versioning/docker/index.spec.ts

```diff
@@ -11,6 +11,24 @@ describe('docker.', () => {
       expect(docker.isValid('3')).toBe('3');
       expect(docker.isValid('foo')).toBeNull();
     });
+    it('it should return null if the version string looks like a git commit hash', () => {
```

Comment:
Let's also have a test for something that's a git commit hash but a number. e.g. 20200801 ?

Reply:
Is this covered by case on L27 ?

Reply:
So in rare cases where commit hash only has numbers, we would treat it as number.

Reply:
Yes, we'd attempt to sort/update it like we do today. Although perhaps there's no chance you get *two* commit hashes both as numbers, so there'd be no update

Reply:
Yes. So for a 7-char hash this is one in 268 million commits? (Please correct my maths if I’m wrong). To avoid this happening entirely the user can use the 40-char hash instead.

Reply:
Actually this maths isn’t quite right but it is pretty unlikely I’m sure.